### DISCUSSION TITLE
Change Keycloak realm in swagger-ui.

### DISCUSSION THREAD

**Discussion by edufolly** [quarkusio/quarkus#22280] [Q&A]
The `quarkus.oidc.auth-server-url` is configured by an environment variable of the container, but `quarkus.smallrye-openapi.oauth2-implicit-token-url` is a configuration property fixed at build time.

Is it possible to change the Keycloak realm in swagger-ui at native mode?

[a comment by an automated account is not shown]

**Comment by phillip-kruger** [comment]
Do you want to pass `quarkus.smallrye-openapi.oauth2-implicit-token-url` in as a runtime property (like setting an env var) and that is not working ?

**Comment by edufolly** [comment]
Yes.

When I build the native image the environment variable is fixed for all pods.

If I leave the variable empty, I get this error:
```> io.quarkus.runtime.configuration.ConfigurationException: SRCFG00011: Could not expand value envvar.realm in property quarkus.smallrye-openapi.oauth2-implicit-token-url```

**Reply by phillip-kruger** [reply]
Ok, we will have to move this to a runtime config I think

**Reply by edufolly** [reply]
Do I have to open an issue?

**Reply by phillip-kruger** [reply]
Yes please

**Reply by edufolly** [reply]
Done. Thanks.
